packages/application/src/test/application.spec.ts

```diff
@@ -0,0 +1,266 @@
+    {
+      cell_type: 'code',
+      execution_count: null,
+      metadata: {},
+      outputs: [],
+      source: ['import os\n', "print(os.environ['HOME'])"]
+    }
+  ],
+  metadata: {
+    kernelspec: {
+      display_name: 'Python 3',
+      language: 'python',
+      name: 'python3'
+    },
+    language_info: {
+      codemirror_mode: {
+        name: 'ipython',
+        version: 3
+      },
+      file_extension: '.py',
+      mimetype: 'text/x-python',
+      name: 'python',
+      nbconvert_exporter: 'python',
+      pygments_lexer: 'ipython3',
+      version: '3.7.6'
+    }
+  },
+  nbformat: 4,
+  nbformat_minor: 4
+};
+const codeSnippetSchema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Code Snippet',
+  name: 'code-snippet',
+  display_name: 'Code Snippet',
+  namespace: 'code-snippets',
+  uihints: {
+    title: 'Code Snippets',
+    icon: 'elyra:code-snippet'
+  },
+  properties: {
+    schema_name: {
+      title: 'Schema Name',
+      description: 'The schema associated with this instance',
+      type: 'string',
+      pattern: '^[a-z][a-z0-9-_]*[a-z0-9]$',
+      minLength: 1
+    },
+    display_name: {
+      title: 'Display Name',
+      description: 'The display name of the Code Snippet',
+      type: 'string',
+      minLength: 1
+    },
+    metadata: {
+      description: 'Additional data specific to this Code Snippet',
+      type: 'object',
+      properties: {
+        description: {
+          title: 'Description',
+          description: 'Code snippet description',
+          type: 'string'
+        },
+        language: {
+          title: 'Language',
+          description: 'Code snippet implementation language',
+          type: 'string',
+          uihints: {
+            field_type: 'dropdown',
+            default_choices: ['Python', 'Java', 'R', 'Scala', 'Markdown']
+          },
+          minLength: 1
+        },
+        code: {
+          title: 'Code',
+          description: 'Code snippet code lines',
+          type: 'array',
+          uihints: {
+            field_type: 'code'
+          }
+        }
+      },
+      required: ['language', 'code']
+    }
+  },
+  required: ['schema_name', 'display_name', 'metadata']
+};
+
+const codeSnippetMetadata = {
+  schema_name: 'code-snippet',
+  display_name: 'tester',
+  name: 'tester',
+  metadata: {
+    language: 'Python',
+    code: ['hello_world']
+  }
+};
+
+const updatedCodeSnippetMetadata = {
+  schema_name: 'code-snippet',
+  display_name: 'tester',
+  name: 'tester',
+  metadata: {
+    language: 'Python',
+    code: ['testing']
+  }
+};
+
+beforeAll(async () => {
+  await server.start();
+});
+
+afterAll(async () => {
+  await server.shutdown();
+});
+
+describe('@elyra/application', () => {
+  describe('FrontendServices', () => {
+    describe('#getSchema', () => {
+      it('should get schema', async () => {
+        expect(await FrontendServices.getSchema('code-snippets')).toEqual([
+          codeSnippetSchema
```

Comment:
ok, so this answers my question about the name, we should use something like `expectedSnippetSchema`